### PR TITLE
CRUD: tenant and project feature flags, approval

### PR DESCRIPTION
**Pull request by yalosev**

### CODE REVIEW

File: vault-plugins/flant_iam/backend/path_identitysharing.go [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+	list, err := repo.List(sourceTenant)
+	if err != nil {
+		return nil, err
+	}
+
+	_ = commit(tx, b.Logger())
+
+	resp := &logical.Response{
+		Data: map[string]interface{}{
+			"uuids": list,
```
Comment (shvgn) [inline vault-plugins/flant_iam/backend/path_identitysharing.go:160]: We should have more verbose objects here

File: vault-plugins/flant_iam/backend/path_rolebindingapproval.go [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+			RequireMFA:            requireMFA,
+			RequireUniqueApprover: requireUniqueApprover,
+		}
+
+		tx := b.storage.Txn(true)
+		defer tx.Abort()
+
+		repo := model.NewRoleBindingApprovalRepository(tx)
+		_, err := repo.GetByID(id)
+		if err != nil {
```
Comment (shvgn) [inline vault-plugins/flant_iam/backend/path_rolebindingapproval.go:155]: This decision making should be in the repo, not in handler. Handler is to parse input and form the response. All `if`'s should be in the repo.

**Review by shvgn — APPROVED** [review]
LGTM

File: vault-plugins/flant_iam/backend/path_identitysharing.go (hunk 1 above)
Comment (yalosev) [reply vault-plugins/flant_iam/backend/path_identitysharing.go]: done

File: vault-plugins/flant_iam/backend/path_rolebindingapproval.go (hunk 2 above)
Comment (yalosev) [reply vault-plugins/flant_iam/backend/path_rolebindingapproval.go]: done

**Pull request merged**